gettsim/visualization.py

```diff
@@ -571,3 +331,246 @@ def _kth_order_successors(dag, node, order):
     if 1 <= order:
         for successor in dag.successors(node):
             yield from _kth_order_successors(dag, successor, order=order - 1)
+
+
+def plot_dag(
+    functions,
+    source_code=False,
+    targets=None,
+    columns_overriding_functions=None,
+    check_minimal_specification="ignore",
+    selectors=None,
+    orientation="v",
+    showlabels=True,
+    showlegend=False,
+):
+    """Plot the dag of the tax and transfer system.
+
+    Parameters
+    ----------
+    functions : str, pathlib.Path, callable, module, imports statements, dict
+        Functions can be anything of the specified types and a list of the same objects.
+        If the object is a dictionary, the keys of the dictionary are used as a name
+        instead of the function name. For all other objects, the name is inferred from
+        the function name.
+    source_code str,
+        Experimental feature which makes the source code of the functions or tbe node
+        names or none accessible as a hover information. Sometimes, the tooltip is not
+        properly displayed.
+    targets : str, list of str
+        String or list of strings with names of functions whose output is actually
+        needed by the user.
+    columns_overriding_functions : str list of str
+        Names of columns in the data which are preferred over function defined in the
+        tax and transfer system.
+    check_minimal_specification : {"ignore", "warn", "raise"}, default "ignore"
+        Indicator for whether checks which ensure the most minimal configuration should
+        be silenced, emitted as warnings or errors.
+    selectors : str or list of str or dict or list of dict or list of str and dict
+        Selectors allow to you to select and de-select nodes in the graph for
+        visualization. For the full list of options, see the tutorial about
+        `visualization <../docs/tutorials/visualize.ipynb>`_. By default, all nodes are
+        shown.
+    orientation :str,default "v"
+        decide whether the graph is horizontal or vertical
+    showlabels : bool, default True
+        Annotate the graph with labels.
```

Comment:
```suggestion
        Whether the graph is annotated with labels.
```
(similar for showlegend)